src/assets/css/icomoon.css

```diff
@@ -1,15 +1,15 @@
 @font-face {
   font-family: 'icomoon';
   src:
-    url('/fonts/icomoon.ttf?ekaf63') format('truetype'),
-    url('/fonts/icomoon.woff?ekaf63') format('woff'),
-    url('/fonts/icomoon.svg?ekaf63#icomoon') format('svg');
+    url('/fonts/icomoon.ttf?kps16g') format('truetype'),
+    url('/fonts/icomoon.woff?kps16g') format('woff'),
+    url('/fonts/icomoon.svg?kps16g#icomoon') format('svg');
   font-weight: normal;
   font-style: normal;
   font-display: block;
 }
 
-[class^="icon-"] {
+[class^="icon-"], [class*=" icon-"] {
```

Comment:
We actually just removed this second selector (in #416), in part for performance reasons and in part because it simplifies other icon-related CSS in `app.scss`.